src/wwwroot/js/site.js

```diff
@@ -1,4 +1,17 @@
-﻿// Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification
-// for details on configuring this project to bundle and minify static web assets.
-
-// Write your JavaScript code.
+﻿$('#btn-comment').click(function() {
+    var d = {

```

Comment:
this is ok, but it think that you can use:
`data: { text: $('#comment-text').val(), id = $('#movie-id').val() }`